backend/clickhouse/query.go

```diff
@@ -39,13 +39,30 @@ func readObjects[TObj interface{}, TReservedKey ~string](ctx context.Context, cl
 	var err error
 	var args []interface{}
 
-	orderForward := OrderForwardNatural
-	orderBackward := OrderBackwardNatural
-	if pagination.Direction == modelInputs.SortDirectionAsc {
-		orderForward = OrderForwardInverted
-		orderBackward = OrderBackwardInverted
+	sortColumn := "timestamp"
+	sortDirection := modelInputs.SortDirectionDesc
+	if params.Sort != nil {
+		sortColumn = params.Sort.Column
+		sortDirection = params.Sort.Direction
 	}
 
+	if col, found := config.KeysToColumns[TReservedKey(sortColumn)]; found {
+		sortColumn = col
+	} else {
+		sortColumn = fmt.Sprintf("%s['%s']", config.AttributesColumn, sortColumn)
```

Comment:
I think we need to make sure sortColumn is used as a parameter instead of interpolated as a string since it's user input e.g. `fmt.Sprintf("%s['%s']", config.AttributesColumn, sb.Var(sortColumn))`

Reply:
Right - makes sense!

cc @SpennyNDaJets since we were going back and forth on this in pairing.